src/test/java/com/google/cloud/spanner/pgadapter/AbstractMockServerTest.java

```diff
@@ -88,7 +88,7 @@
  * server received the requests that the test expected.
  */
 public abstract class AbstractMockServerTest {
-  private static final Logger logger = Logger.getLogger(AbstractMockServerTest.class.getName());
+  protected static final Logger logger = Logger.getLogger(AbstractMockServerTest.class.getName());
```

Comment:
nit: loggers should normally be kept private and specific to one class, and a new logger should be recreated in the subclass. This makes it easier to configure loggers per class.

Reply:
Done